policyengine_canada/parameters/gov/provinces/mb/benefits/mbcb/base.yaml

```diff
@@ -0,0 +1,15 @@
+description: Manitoba provides this base amount under the Child Tax Benefit.
+values:
+  2022-01-01: 420 # 35 per month
+
+metadata:
+  unit: currency-CAD
+  label: Manitoba child benefit base
+  period: year
+  reference:
+    - title: Manitoba child tax benefit 
+      href: https://www.gov.mb.ca/fs/eia/mcb.html
+    - title: Manitoba child benefit regulation § 6(1)
+      href: https://www.gov.mb.ca/fs/eia_manual/regs/4.html
+    - title: Manitoba child benefit regulation § 16
+      href: https://www.gov.mb.ca/fs/eia_manual/regs/4.html
```

Comment:
```suggestion
```

Reply:
dont need this 

- is there a tax form for 2022 that we can reference? If not please leave a comment

Same for all yaml